src/inspector/view/qml/MuseScore/Inspector/general/playback/PlaybackGeneralSettings.qml

```diff
@@ -47,62 +47,115 @@ Item {
 
         spacing: 12
 
+        property var fullModel: [
+            { typeRole: Inspector.TYPE_NOTE, componentRole: noteSection },
+            { typeRole: Inspector.TYPE_ARPEGGIO, componentRole: arpeggioSection },
+            { typeRole: Inspector.TYPE_FERMATA, componentRole: fermataSection },
+            { typeRole: Inspector.TYPE_BREATH, componentRole: pausesSection },
+            { typeRole: Inspector.TYPE_GLISSANDO, componentRole: glissandoSection },
+            { typeRole: Inspector.TYPE_GRADUAL_TEMPO_CHANGE, componentRole: tempoChangeSection }
+        ]
+
+        function getVisibleModel() {
+            var visibleModel = [];
+            var currType = null;
+            for (let i = 0; i < fullModel.length; i++) {
+                currType = root.proxyModel ? root.proxyModel.modelByType(fullModel[i].typeRole) : null;
```

Comment:
1. It's better to declare currType inside the loop:

var currType = root.proxyModel ? root.proxyModel.modelByType(fullModel[i].typeRole) : null

2. its name should be itemModel (or something similar) to avoid confusing it with some enums

3. it's better to use modelByType only once. Just add the found model as a property to visibleModel (so we won't have to call modelByType again inside Loader)